tools/integration/src/index.ts

```diff
@@ -271,11 +288,198 @@ yargs
+    }
+    try {
+        await validatePackageJson(packageData, errors, warnings, successMessages);
+        validateDashboardFiles(dashboardsPath, errors, warnings, successMessages);
+    } catch (error) {
+        errors.push(`Linting failed: ${error}`);
+    }
+
+    // Check if debug is enabled
+    const isDebug = argv.debug;
+
+    if (isDebug) {
+	successMessages.forEach((message) => {
+            logger.info(message);
+        });
+        if (warnings.length > 0) {
+            logger.warn(`Warnings encountered during linting:`);
+            warnings.forEach((warning) => {
+                logger.warn(warning);
+            });
+        }
+        if (errors.length > 0) {
+            logger.error(`Linting failed with the following errors:`);
+            errors.forEach((error) => {
+                logger.error(error);
+            });
+        }
+    }
+
+    if (errors.length > 0) {
+        logger.error(`Linting failed.`);
+        process.exit(-1);
+    } else {
+        if (!isDebug) {
+            logger.info('Linting completed successfully.');
+        }
+        process.exit(0);
+    }
+}
+
+// Helper function to validate `package.json`
+async function validatePackageJson(packageData: any, errors: string[], warnings: string[], successMessages: string[]): Promise<void> {
+    // Validate `name`
+    const namePattern = /^@instana-integration\/[a-zA-Z0-9-_]+$/;
+    if (!namePattern.test(packageData.name)) {
+        warnings.push(`Warning: Package name "${packageData.name}" does not align with the IBM package naming convention.`);
+    } else {
+        successMessages.push('Field "name" is valid.');
+    }
+
+    // Validate `version`
+    const versionPattern = /^\d+\.\d+\.\d+$/;
+    if (!versionPattern.test(packageData.version)) {
+        errors.push(`Invalid version "${packageData.version}". The version must follow the format "x.x.x".`);
+    } else {
+        successMessages.push('Field "version" format is valid.');
+    }
+
+    // Fetch the currently published version from npm and compare
+    try {
+        const response = await axios.get(`https://registry.npmjs.org/${packageData.name}`);
+        const publishedVersion = response.data['dist-tags']?.latest || '1.0.0';
+
+        if (semver.eq(packageData.version, publishedVersion)) {
+            errors.push(`Package version "${packageData.version}" is the same as the currently published version "${publishedVersion}". It must be greater than the currently published version`);
+        } else if (semver.lt(packageData.version, publishedVersion)) {
+            errors.push(`Invalid version "${packageData.version}". It must be greater than the currently published version "${publishedVersion}".`);
+        } else {
+            successMessages.push('Package version is valid and greater than the currently published version.');
+        }
+    } catch (error) {
+        if ((error as AxiosError).response?.status === 404) {
+	    if (packageData.version === '1.0.0') {
```

Comment:
If it's a new package, can we just leave as is w/o restricting the init version?